packages/grafana-ui/src/components/FormField/FormField.tsx

```diff
@@ -36,13 +36,15 @@ export const FormField = ({
   ...inputProps
 }: Props) => {
   const styles = getStyles();
+  const id = useId();
   return (
     <div className={cx(styles.formField, className)}>
-      <InlineFormLabel width={labelWidth} tooltip={tooltip} interactive={interactive}>
+      <InlineFormLabel htmlFor={id} width={labelWidth} tooltip={tooltip} interactive={interactive}>
         {label}
       </InlineFormLabel>
       {inputEl || (
         <input
+          id={id}
```

Comment:
this only fixes it for the case where an `inputEl` isn't passed - but since the component is deprecated anyway that feels good enough?